### ISSUE TITLE
How to load multi model files? 

### ISSUE THREAD

**Issue by woshichuanqilz** [OlafenwaMoses/ImageAI#151]
I have many model files, can I load some of them once ? 

I check the `setModelPath` function seems not support a list input as files.

```
    def setModelPath(self, model_path):
        """
        'setModelPath()' function is required and is used to set the file path to the model adopted from the list of the
        available 4 model types. The model path must correspond to the model type set for the prediction instance object.

        :param model_path:
        :return:
        """
        self.modelPath = model_path
```

**Comment by OlafenwaMoses** [comment]
See this article for latest update on this.

https://towardsdatascience.com/the-story-and-future-of-imageai-one-year-anniversary-e63c80f527c8

**Comment by OlafenwaMoses** [comment]
@woshichuanqilz  Thanks very much for your patience. We have released a new version (v2.0.3) that allows you to run multiple custom models at once. See the link below on this.

https://github.com/OlafenwaMoses/ImageAI/blob/master/imageai/Prediction/CUSTOMPREDICTION.md#custompredictionmultiple

Also visit https://github.com/OlafenwaMoses/ImageAI to learn more on all the new features.
